docs/content/practices/state_clearing.md

```diff
@@ -118,15 +119,15 @@ It's just a shell client and should be installed to the device.
     Despite the fact that it does the job, this solution looks overcomplicated:
 
     1. We need to install +2 different apk to each emulator
-    2. We delegate this job to the device instead of host machine. 
+    2. We delegate this job to the device instead of the host machine. 
     <br/>Devices are less reliable than host pc
```

Comment:
Less reliably in the sense of "they do not lose connection to adb server, issues with battery, internet, system dialogs," or sth like that?